### PR TITLE
Another onboarding modal variant!

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

screen shot 2016-08-26 at 1 08 55 pm

Still needs a design pass form Luke (feature card could use some tweaking)
Also not mobile ready, just here so we can run an optimizely test & check how it performs. If it does well we can delete all of these variants & redo the onboarding CSS to be prettier + be mobile compliant
#### How should this be reviewed?

Add -modal to the onboarding div as a new class
#### Relevant tickets

Another modal for #6929 
#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Onboarding extends React.Component {
     }
 
     return (
-      <div id="onboarding" className="onboarding">
+      <div id="onboarding" className="onboarding -modal">
```
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js]: needs to be removed before merge

File: lib/themes/dosomething/paraneue_dosomething/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@dosomething/forge": "~6.7.0",
-    "dosomething-modal": "~0.3.0",
+    "dosomething-modal": "0.3.3",
```
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/package.json:22]: the hell... https://github.com/DoSomething/phoenix/blob/dev/lib/themes/dosomething/paraneue_dosomething/package.json#L22

File: lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js (hunk 1 above)
Comment (weerd) [reply lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js]: Oh cause this is getting added via Optimizely?

File: lib/themes/dosomething/paraneue_dosomething/package.json (hunk 2 above)
Comment (weerd) [reply lib/themes/dosomething/paraneue_dosomething/package.json:22]: Did you pull latest `dev` and rebase?

File: lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js (hunk 1 above)
Comment (itsjoekent) [reply lib/themes/dosomething/paraneue_dosomething/js/components/Onboarding.js]: yep!

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss [hunk 3]
```diff
@@ -43,6 +43,64 @@
...
+        margin-left: 7.5%;
+      }
+
+      .beta-card {
+        margin-top: -32px;
+        box-shadow: $box-shadow;
+      }
+
+      .photo.-framed {
+        box-shadow: 0 2px 12px rgba(0, 0, 0, 0.3);
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: use `$box-shadow`

**Comment by weerd** [comment]
Have you checked this in Firefox, Safari and IE9+ apart from Chrome? Does it all look cool?

**Comment by itsjoekent** [comment]
@weerd Nope!

**Comment by weerd** [comment]
@deadlybutter Fendering essentials! Use Browserstack for any browser you need to test that you don't have installed on your computer (eg: IE)

**Comment by itsjoekent** [comment]
@weerd having problems with IE, but firefox, chrome & safari all seem 🆗  :D

**Comment by itsjoekent** [comment]
not sure if its browserstack or IE, cause im running those 3 just natively on the computer

**Comment by weerd** [comment]
👍 

Needs IE testing when on staging!

**Pull request merged**